Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -97,19 +97,16 @@ Describe "Test importing correct customized rules" {
 		    $customizedRulePath.Count | Should Be 1
 		}
 
-        It "will show the custom rule when given a rule folder path with trailing backslash" {
+        It "will show the custom rule when given a rule folder path with trailing backslash" -skip:(!$IsWindows){
```

Comment:
I guess you are planning to introduce a `Travis` build for Linux/Mac then?

Reply:
yup, that's the plan, I'll work on that issue

Reply:
Thanks. Looks good to me.